modules/sildisco/public/metadata.php

```diff
@@ -189,16 +188,16 @@
     if ($idpmeta->hasValue('contacts')) {
         $contacts = $idpmeta->getArray('contacts');
         foreach ($contacts as $contact) {
-            $metaArray['contacts'][] = Metadata::getContact($contact);
+            $metaArray['contacts'][] = $metadataUtils->getContact($contact);
         }
     }
 
-    $technicalContactEmail = $config->getString('technicalcontact_email', false);
-    if ($technicalContactEmail && $technicalContactEmail !== 'na@example.org') {
+    $technicalContactEmail = $config->getOptionalString('technicalcontact_email', null);
```

Comment:
Just curious, is there is a default of null, such that the second parameter is overkill?

Reply:
Good question. The signature is:

```php
public function getOptionalString(string $name, ?string $default): ?string
```

I think the second parameter is required.